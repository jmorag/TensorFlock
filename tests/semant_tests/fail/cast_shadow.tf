main = cast 2;
cast : Nat -> Nat;
cast n = n + 3;
