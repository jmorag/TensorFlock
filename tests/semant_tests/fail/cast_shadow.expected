Fatal error: exception Failure("attempting to redefine already defined symbol: cast")
